manifests/s/SeventhString/Transcribe/0.9.0.20/SeventhString.Transcribe.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: SeventhString.Transcribe
+PackageVersion: 0.9.0.20

```

```suggestion
PackageVersion: 9.20
```
This must match control panel; Remember to update the name of the version folder when updating this field